crates/hyperqueue/src/client/commands/journal/output.rs

```diff
@@ -76,32 +76,26 @@ fn format_payload(event: EventPayload) -> serde_json::Value {
                 "allocation-id": allocation_id,
             })
         }
-        EventPayload::TaskStarted {
-            job_id, task_id, ..
-        } => json!({
+        EventPayload::TaskStarted { task_id, .. } => json!({
```

Comment:
Maybe we should rename `task_id` fields here to just `id`?

Reply:
I would prefer, to have "task_id", because in many contexts we are having several ids at once, worker_id, job_id, task_id etc, and having explicit "task_id" seems to more clear while it is still a short name.

Reply:
Ok, fair enough.